CMakeLists.txt

```diff
@@ -26,6 +27,7 @@ add_subdirectory(seedimg)
 add_subdirectory(seedimg-jpeg)
 add_subdirectory(seedimg-png)
 add_subdirectory(seedimg-webp)
+add_subdirectory(seedimg-format-farbfeld)
```

Comment:
make it seedimg-farbfeld to adhere with naming scheme

Reply:
what's bad with this '-'

Reply:
it doesnt adhere to naming scheme '-'

Reply:
i chang the whol schem